recipes/napari-segment-blobs-and-things-with-membranes/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-segment-blobs-and-things-with-membranes" %}
```

Comment:
This name 😂

Reply:
I bet, nobody will ever ask: What does this plugin do? 😉😅🙌